sydent/pipeline.yml

```diff
@@ -18,11 +18,83 @@ steps:
       - "pip install git+https://github.com/matrix-org/matrix-is-tester.git"
       - "echo '+++ Run Tests'"
       - "trial matrix_is_tester"
-    label: ":pytest: matrix_is_tester"
+    label: ":pytest: matrix_is_tester / :python: 2.7"
     timeout_in_minutes: 5
     plugins:
       - docker#v3.0.1:
           image: "python:2.7"
     artifact_paths:
       - "matrix_is_test/sydent.stderr"
       - "_trial_temp/test.log"
+
+  - command:
+      - "echo '--- Install dependencies'"
+      - "pip install -e ."
+      - "echo '+++ Run unit tests'"
+      - "trial tests"
```

Comment:
it'd sure be swell to stick these 4 lines in a script so that we don't need to have 3 copies of them here. Likewise the ones for the IS tester.